demo/demo.cpp

```diff
@@ -162,7 +162,9 @@ int WispEntry()
 
 	render_system->InitSceneGraph(*scene_graph.get());
 
-	fg_manager::Setup(*render_system, &RenderEditor);
+	bool is_fallback = wr::d3d12::GetRaytracingType(render_system->m_device) == wr::RaytracingType::FALLBACK;
```

Comment:
The render system is already passed to the setup function, no need to get the is_fallback variable here.